components/Header/Artist.tsx

```diff
@@ -46,29 +47,45 @@ const Artist = ({
 
   return (
     <button
-      className={`${
+      className={cn(
+        "py-2 w-full outline-none",
         isMini
-          ? `${isSelectedArtist && "w-fit rounded-full"} flex justify-center items-center`
-          : `flex gap-3 items-center px-2 text-sm rounded-md text-grey-dark ${isAnyArtistSelected ? 'hover:bg-grey-light-1' : ''} ${isSelectedArtist && "!bg-grey-light-1"}`
-      } py-2 w-full ${shouldHighlight ? 'z-50 relative' : ''}`}
+          ? [
+              "flex justify-center items-center",
+              isSelectedArtist && "w-fit rounded-full"
+            ]
+          : [
+              "flex gap-3 items-center px-2 text-sm rounded-md text-grey-dark",
+              isAnyArtistSelected && "hover:bg-grey-light-1",
+              isSelectedArtist && "!bg-primary/10"
+            ],
+        shouldHighlight && "z-50 relative"
+      )}
```

Comment:
Classes are the same as earlier, but these are now arranged using tailwind-merge `cn` util function.

Changes in <button>:
```
Before: isSelectedArtist && "!bg-grey-light-1"
After: isSelectedArtist && "!bg-primary/10"

Added outline-none for unusual focus fix.
```